ecs-cli/modules/cli/compose/entity/service/service.go

```diff
@@ -475,6 +490,19 @@ func (s *Service) GetTags() ([]*ecs.Tag, error) {
 	return s.tags, nil
 }
 
+func (s *Service) getLoadBalancerList(loadBalancerValue []string) ([]*ecs.LoadBalancer, error) {
+	if s.loadBalancers == nil {
+		loadBalancers := make([]*ecs.LoadBalancer, 0)
+		var err error
+		loadBalancers, err = utils.ParseLoadBalancers(loadBalancerValue, loadBalancers)
+		if err != nil {
+			return nil, err
+		}
+		s.loadBalancers = loadBalancers
```

Comment:
This is confusing, since you also assign value to `s.loadBlancers` outside of this function. It ends up being `s.LoadBalancers = s.LoadBalancers`.
 
I think you probably don't need `getLoadBalancerList` and let `utils.ParseLoadBalancers` takes the slice of string as input and slice of struct as output.

Reply:
Yeah I like that idea a lot better!